.github/actions/setup-relenv/action.yml

```diff
@@ -22,7 +22,7 @@ inputs:
     required: false
     type: string
     description: The version of relenv to use
-    default: 0.10.2
+    default: 0.12.0
```

Comment:
Why are these first 3 files being kept at 0.12.0 instead of 0.12.1 like the others?

Reply:
Just because they weren't updated. FYI, those are the default values if no value is passed in(we always pass values)

Now that's no longer a real problem since I added checks to make sure that the relenv being used is the one we actually target.
Previously, not updating those files could mean that we would be using the wrong(older) version of relenv.

But again, those are the default value. It's a good hygiene to also update them, but it's not going to break anything if we don't.

Reply:
Fixed